velocity.py

```diff
@@ -1,33 +1,31 @@
-from tkinter import Tk
+from tkinter import *
 from tkinter import Frame
 from tkinter import LabelFrame
 from tkinter import Text
 from tkinter import Entry
 from tkinter import NORMAL
 from tkinter import DISABLED
+from tkinter.messagebox import showinfo
 from collections import deque
-#develop branch
-window = Tk()
-window.title("VELOCITY - speed + direction")
-window.geometry("600x400")
-window.minsize(width=200, height=200)
 
-text_frame = Frame(window, height=30, width=30, highlightbackground="black", highlightthickness=1)
-text_frame.pack(pady=12)
-editor_frame = LabelFrame(window, padx=2, pady=2, bd=0, text="Input your text here")
-editor_frame.pack(pady=10)
+def new_file():
```

Comment:
need discussion over this.

Reply:
We don't need any **new file**. Actually we don't even need any menu bar, we need just a button so that we can select different paragraphs.